### PR TITLE
Rename some variables

### PR DESCRIPTION
**Pull request by skorokithakis**

### CODE REVIEW

**Comment by skorokithakis** [comment]
I've renamed some things, I thought `ip_addr` looks more Pythonic with an underscore between the words, so I renamed that.

What do you think about creating getters for `name`, `ip_addr` and `uuid`? That will make it much easier to document, although I'm not sure if we can document instance variables with things like Sphinx. If we can, we should do it that way.

**Comment by skorokithakis** [comment]
Closes #152, by the way.

**Comment by theychx** [comment]
> What do you think about creating getters for name, ip_addr and uuid?

I would prefer ```@property``` decorated methods.

**Comment by skorokithakis** [comment]
> I would prefer `@property` decorated methods.

That's what I meant by getters, vs just `self.name = self._cast.name`.

File: catt/api.py [hunk 1]
```diff
@@ -11,56 +11,60 @@
...
+        self.ip_addr = self._cast.host
+        self.uuid = self._cast.uuid
 
-    def _create_ctrl(self) -> None:
+    def _create_controller(self) -> None:
         self._cast_controller = get_controller(self._cast, get_app_info("default"))
 
     @property
-    def ctrl(self):
+    def controller(self):
```
Comment (theychx) [inline catt/api.py]: I would prefer ```control``` over ```controller```, as it's more distinct from the slightly confusing controller/castcontroller "paradigm".

Comment (skorokithakis) [reply catt/api.py]: `control` is a verb though, implying a function. Or does it return "a control"?

**Comment by theychx** [comment]
Ah, ok. No problem then.

**Comment by skorokithakis** [comment]
There's no problem with that, right? It doesn't fetch the name through the network or something?

File: catt/api.py (hunk 1 above)
Comment (theychx) [reply catt/api.py]: > Or does it return "a control"?

That is what I meant, I see your point though, about it being a verb. I just dont' like ```controller```.

Comment (skorokithakis) [reply catt/api.py]: Yeah... We can find a completely different name, it doesn't have to be `controller`.

**Comment by theychx** [comment]
> It doesn't fetch the name through the network or something?

Nope, only the one time the ```Chromecast``` object is created.

**Comment by skorokithakis** [comment]
Thought so, so we're good to make these properties. I think we're good to leave it for now and only change it when it becomes a documentation tooling problem.

**Comment by skorokithakis** [comment]
Merged, thanks!

**Comment by theychx** [comment]
Thank you!

**Comment by theychx** [comment]
> Closes #152, by the way.

Github magic didn't close the issue you opened. Is it the comma? Probably just a gh brainfart.

**Pull request merged**